Corona-Warn-App/src/main/java/de/rki/coronawarnapp/contactdiary/storage/entity/ContactDiaryLocationEntity.kt

```diff
@@ -19,10 +20,13 @@ data class ContactDiaryLocationEntity(
         get() = locationId
 }
 
+private const val MAX_CHARACTERS = 250
+private fun String.trimMaxCharacters(): String = this.trimToLength(MAX_CHARACTERS)
```

Comment:
`trimToLength` also removes leading and trailing white spaces

Reply:
I checked it, it uses substring internally, so it doesn't remove them ...

Reply:
The ViewModel and the Fragment did some trimming too, that's why it was still trimmed 🙃